src/Package.php

```diff
@@ -43,6 +50,7 @@ class Package {
 	public static function init() {
 		$wordpress_minimum_met = version_compare( get_bloginfo( 'version' ), '5.3', '>=' );
```

Comment:
Not directly from this PR obviously, but I feel the check here should use `get_dependency_errors()` instead of creating required version numbers in multiple places.

Reply:
Good idea! Done in https://github.com/woocommerce/woocommerce-admin/pull/3851/commits/a75cfbb64c6d20db3378a030869a83603fe5fdb7